docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md

```diff
@@ -19,11 +19,13 @@ YugabyteDB Anywhere supports deploying YugabyteDB on both x86 and ARM (aarch64)
 
 YugabyteDB Anywhere supports deploying YugabyteDB on the following operating systems:
 
-* AlmaLinux OS 8
-* CentOS 7
-* Oracle Linux 8
-* Ubuntu 18
-* Ubuntu 20
+* CentOS (default)
```

Comment:
Btw the default w 2.15.3+ is AlmaLinux OS 8

Reply:
Marked AlmaLinux 8 as default; Deleted "tested"